crates/aptos-ledger/src/lib.rs

```diff
@@ -0,0 +1,301 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use hex::{encode, decode};
+use ledger_apdu::APDUCommand;
+use ledger_transport_hid::{hidapi::HidApi, TransportNativeHID};
+use std::str;
+use bcs::to_bytes;
+
+const DERIVATIVE_PATH: &str = "m/44'/637'/0'/0'/0'"; // TODO: Add support for multiple index
+
+const CLA_APTOS: u8 = 0x5B; // Aptos CLA Instruction class
+const INS_GET_VERSION: u8 = 0x03; // Get version instruction code
+const INS_GET_APP_NAME: u8 = 0x04; // Get app name instruction code
+const INS_GET_PUB_KEY: u8 = 0x05; // Get public key instruction code
+const INS_SIGN_TXN: u8 = 0x06; // Sign the transaction
+const APDU_ANSWER_CODE: u16 = 36864; // success code for transport.exchange
+
+const MAX_APDU_LEN: u16 = 255;
+const CHUNK_SIZE: usize = 128; // Chunk size to be sent to Ledger
+const P1_NON_CONFIRM: u8 = 0x00;
```

Comment:
21, 23 and 25 have the same value, maybe unify?

Reply:
Ah yeah, I was hestating unifying them. But each one serve a different purpose, so I decided to separate them out. (Same format as the Petra wallet - ledger integration)